app/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,53 @@
+import cx from "classnames";
+
+import styles from "./Progress.module.scss";
+
+type ProgressProps = {
+  firstDose: number;
+  secondDose: number;
+  size?: "small" | "large";
+  goal?: number;
+};
+
+export const Progress = ({
+  firstDose,
+  secondDose,
+  size = "small",
+  goal = 0.9,
```

Comment:
I've updated these props to expect values less than 1 - so that they match the data store. IMO the less we manipulate on the way through the better!